documentation/index.md

```diff
@@ -64,23 +64,61 @@ notifications the first time will prompt the user to enable notifications.
 Sets or gets the channel tags. Tags can be used to segment the audience.
 
 ```
-    Airship.tags = ["test", "titanium"];
+    Airship.tags = ["test", "titanium"]
 
     Airship.tags.forEach(function(tag) {
-        Ti.API.info("Tag: " + tag);
+        Ti.API.info("Tag: " + tag)
     });
 ```
 
+
+
 #### namedUser
 
 Sets the namedUser for the device.
 
 ```
-    Airship.namedUser = "totes mcgoats";
+    Airship.namedUser = "totes mcgoats"
 ```
 
 ## Methods
 
+#### createChannelTagsEditor
+
+The tag editor allows adding and removing tags on the channel.
+
+```
+    var editor = Airship.createChannelTagsEditor()
+    editor.clearTags()
+    editor.addTags("neat", "rad")
+    editor.removeTags("cool")
+    editor.applyTags()
+```
+
+#### createChannelTagGroupEditor
+
+The tag editor allows editing channel tag groups.
+
+```
+    var editor = Airship.createChannelTagGroupsEditor()
```

Comment:
Probably my Titanium-ignorance, but where is `createChannelTagGroupsEditor` defined?

Reply:
Its magically created for you if you names things just right.

Reply:
I like magic.